Cargo.toml

```diff
@@ -41,6 +41,7 @@ log = "0.4"
 itertools = "0.10"
 hex = { version = "0.4", features = ["std"] }
 zstd = "0.11.2"
+zstd-sys = "=2.0.1+zstd.1.5.2"
```

Comment:
Zstd crate seems to have issues currently, hence workaround

https://github.com/gyscos/zstd-rs/issues/177

Reply:
Could you add this as inline comment?